### PR TITLE
Autoloader / Watcher

### PR DESCRIPTION
**Pull request by kengoldfarb**
* Generate autoloader files
* Watch command
  * Known bug: Control is not passed back from inquirer -> readline after running a command that has a prompt

### CODE REVIEW

File: packages/spruce-cli/src/commands/Remote/RemoteCommand.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export default class RemoteCommand extends AbstractCommand {
 	public async setEnvironment(
 		environmentParam?: RemoteStoreRemoteType | string
 	) {
+		console.log('************ setEnvironment')
```
Comment (ryanrd) [inline packages/spruce-cli/src/commands/Remote/RemoteCommand.ts:21]: did you mean to leave this in?

Comment (kengoldfarb) [reply packages/spruce-cli/src/commands/Remote/RemoteCommand.ts]: Nope. Removed

File: packages/spruce-cli/src/commands/test.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export default {
+	this: 'object',
+	default: 'ITS THE DEFAULT'
```
Comment (ryanrd) [inline packages/spruce-cli/src/commands/test.ts:3]: IT IS or  IT'S - unless "THE DEFAULT" belongs to IT

Comment (kengoldfarb) [reply packages/spruce-cli/src/commands/test.ts]: File removed

**Pull request merged**